ci_scripts/run_builds.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+function info {
+  echo "[$(basename "${0}")] [INFO] ${1}"
+}
+
+function die {
+  echo "[$(basename "${0}")] [ERROR] ${1}"
+  exit 1
+}
+
+# Verify xcpretty is installed
+if ! command -v xcpretty > /dev/null; then
+  if [[ "${CI}" != "true" ]]; then
+    die "Please install xcpretty: https://github.com/supermarin/xcpretty#installation"
+  fi
+
+  info "Installing xcpretty..."
+  gem install xcpretty --no-ri --no-rdoc || die "Executing \`gem install xcpretty\` failed"
+fi
+
+# Install sample app dependencies
+info "Installing sample app dependencies..."
+
+cd "Example" || die "Executing \`cd\` failed"
+carthage bootstrap --platform iOS
+carthage_exit_code="$?"
+cd .. || die "Executing \`cd\` failed"
+
+if [[ "${carthage_exit_code}" != 0 ]]; then
+  die "Executing carthage failed with status code: ${carthage_exit_code}"
+fi
+
+# Execute sample app builds (iPhone 6, iOS 11.x)
+info "Executing sample app builds (iPhone 6, iOS 11.x)..."
+
+xcodebuild build \
+  -workspace "Stripe.xcworkspace" \
+  -scheme "Standard Integration (Swift)" \
+  -sdk "iphonesimulator" \
+  -destination "platform=iOS Simulator,name=iPhone 6,OS=11.2" \
+  | xcpretty
```

Comment:
Was it intentional to drop the `-c` flag that colorizes the output?

Reply:
Yeah, to me, the default behavior seems to works so why bother specifying it:

<img width="907" alt="screen shot 2018-07-09 at 3 05 30 pm" src="https://user-images.githubusercontent.com/28276156/42478429-8b94d32e-8389-11e8-8fd5-c4c862faf7a2.png">